speach_relay/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM maven:3.6-jdk-11-openj9
+
+WORKDIR /usr/src/app
+
+COPY src/ ./src/
+COPY jetty* ./
```

Comment:
Is this a generated directory? If so, we probably want to build it here so the build is reproducible.

Reply:
those are configs